projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/GraphQLJobExecutionsSortByEndTimeIT.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service;
+
+import com.vmware.taurus.RepositoryUtil;
+import com.vmware.taurus.ServiceApp;
+import com.vmware.taurus.service.deploy.DeploymentService;
+import com.vmware.taurus.service.model.*;
+import org.junit.jupiter.api.*;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest(classes = ServiceApp.class)
+@AutoConfigureMockMvc
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class GraphQLJobExecutionsSortByEndTimeIT {
```

Comment:
This test looks like integration test. What do you think? @mrMoZ1, @tpalashki

Reply:
That was the idea. Is this not the correct place? The test makes real calls to the graphQl and runs quickly it also exercises the sort and pagination logic.

Reply:
Yes, it should be under the `src/integration-test`.

Reply:
It depends on a method in the com.vmware.taurus.RepositoryUtil class in the testing classes which means that I will have to copy/paste this method in the IT test directory. I also did see that we have other IT tests in the same package so decided to put it there. Should I move it to the integration-test directory?

Reply:
You can easily replace it with jobExecutionRepository.save().

Reply:
moved test to integration-test directory